### PR TITLE
fixed the issue where adding an image that's not integration image as…

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34284

## Description
Fixed the issue where adding an image that's not integration image asks for a RN and a new version.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
What will happen for documentation images?

**Review by yaakovi — CHANGES_REQUESTED** [review]
add unit-tests:
* for find_type - an image named `Author_image.png` returned the expected type.
* for validate manager, added/modified author image doesn't fail the release notes validation.

**Comment by yaakovi** [comment]
> @merit-maita @yaakovi What will happen for documentation (README) images?

doc images have their own file type DOC_IMAGE. their behavior should not be affected

**Comment by merit-maita** [comment]
@ShahafBenYakir 
as @yaakovi mentioned, one more thing, every image that does not end with _image.png, is considered as DOC_IMG, and skipped automatically, as we saw in Guy's screenshot, and it exists in the DOC_FILES folder thats also ignored. the reason why the build you mentioned failed was because it ended with _image.png, so it was considered as an image that has an effect, but it's Author_image, so it had to be excluded

File: demisto_sdk/commands/common/tests/tools_test.py [hunk 1]
```diff
@@ -105,7 +105,8 @@ def test_get_dict_from_file(self, path, _type):
         (VALID_SCRIPT_PATH, FileType.SCRIPT),
         (VALID_WIDGET_PATH, FileType.WIDGET),
         (IGNORED_PNG, None),
-        ('', None)
+        ('', None),
+        ('Author_image.png', None)
```
Comment (yaakovi) [inline demisto_sdk/commands/common/tests/tools_test.py:109]:
```suggestion
        ('Author_image.png', None),
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**